src/style.css

```diff
@@ -0,0 +1,20 @@
+body {
+  font-size: 13px;
+  font-family: Arial;
+}
+
+.h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
+}
+
+.h2 {
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice